tensorflow_encrypted/config.py

```diff
@@ -247,9 +246,9 @@ def session(
         master: Optional[Union[int, str]] = None,
         log_device_placement: bool = False
     ) -> tf.Session:
-        cpu_cores = get_cpu_quota()
+        cpu_cores = get_docker_cpu_quota()
```

Comment:
is this enough to only specify the properties if running in docker? seems like the protobuf file gives default integer values..?

Reply:
Not sure to follow, concerning TF config, if `None` (or no values) are provided, TF will optimize them itself.

Reply:
could we avoid setting them instead of passing None? or are we sure that's doing what we expect?

Reply:
Interesting fact: the default value is 0 since it's should be a `int`.

I'll this piece of code so we don't set it with `None` then